APIs_RTOS/ThisThread_Blinky/main.cpp

```diff
@@ -0,0 +1,23 @@
+#include "mbed.h"
+#include "rtos.h"
+
+#define STOP_FLAG 1
+
+// Toggles LED until the STOP_FLAG is set
```

Comment:
```suggestion
// Toggles LED
```
What I removed is an implementation detail.